yt_dlp/extractor/kaltura.py

```diff
@@ -386,10 +390,35 @@ def _real_extract(self, url):
 
         mobj = self._match_valid_url(url)
         partner_id, entry_id, player_type = mobj.group('partner_id', 'id', 'player_type')
+        flashvars = smuggled_data.get('flashvars')
         ks, captions = None, None
         if not player_type:
             player_type = 'kwidget' if 'html5lib/v2' in url else 'html5'
-        if partner_id and entry_id:
+        if flashvars:
+            flashvars = self._parse_json(flashvars, entry_id)
+            params = {'wid': f'_{partner_id}', 'entry_id': entry_id, 'uiconf_id': smuggled_data.get('uiconf_id')}
+            for k, v in flashvars.items():
+                params['flashvars[' + k + ']'] = json.dumps(v) if not isinstance(v, str) else v
+            webpage = self._download_webpage(update_url_query(
+                'https://cdnapisec.kaltura.com/html5/html5lib/v2.98/mwEmbedFrame.php',
+                params), entry_id)
```

Comment:
For now, the version number is "2.101". Access the URL for extracting `service_mobj` and we can get this:

```js
window['MWEMBED_VERSION'] = '2.101';
window['SCRIPT_LOADER_URL'] = 'https://cdnapisec.kaltura.com/html5/html5lib/v2.101/load.php';

```

.